docs/sql-ref-syntax-aux-show-views.md

```diff
@@ -99,7 +99,7 @@ SHOW VIEWS FROM default LIKE 'sam*';
 | default   | sam1       | false        |
 +-----------+------------+--------------+
 
--- List all views from the current database matching the pattern `sam|suj｜temp*`
+-- List all views from the current database matching the pattern `sam|suj|temp*`
```

Comment:
Not related to this PR, but I would like to take this opportunity to bring up this `correction`. (Full angle to half angle)